csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase.cs

```diff
@@ -3,17 +3,14 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Provides the base class from which the classes that represent all segments walker are derived. This is an abstract class.</para>
+    /// <para>Предоставляет базовый класс, от которого наследуются классы, реализующие AllSegmentsWalkerBase. Это абстрактный класс.</para>
```

Comment:
Lol. Review of the Russian text in English language 
```suggestion
    /// <para>Базовый абстрактный класс для всех [вставьте тут синоним слова "волкеров"]</para>
```
Use smaller doc-comments
No extra information is needed


_The subjective thinking of a C++fan_